docs/wip/demo_dapr.md

```diff
@@ -0,0 +1,295 @@
+---
+title: "Integrate Dapr with OSM"
+description: "A simple demo showing to integrate Dapr with OSM"
+type: docs
+---
+
+This document walks you through the steps of getting Dapr working with OSM on a kubernetes cluster.
+
+1. Install Dapr on your cluster with mTLS disabled:
+
+      1. Dapr has a quickstart repository to help users get familiar with dapr and its features. For this integration demo we will be leveraging the [hello-kubernetes](https://github.com/dapr/quickstarts/tree/master/hello-kubernetes) quickstart. As we would like to integrate this Dapr example with OSM, there are a few modifications required and they are as follows:
+
+         - The [hello-kubernetes](https://github.com/dapr/quickstarts/tree/master/hello-kubernetes) demo installs Dapr with mtls enabled (by default), we would **not want mtls from Dapr and would like to leverage OSM for this**. Hence while [installing Dapr](https://github.com/dapr/quickstarts/tree/master/hello-kubernetes#step-1---setup-dapr-on-your-kubernetes-cluster) on your cluster, make sure to disable mtls by passing the flag : `--enable-mtls=false`  during the installation
+         - Futher [hello-kubernetes](https://github.com/dapr/quickstarts/tree/master/hello-kubernetes) sets up everything in the default namespace, it is **strongly recommended** to set up the entire hello-kubernetes demo in a specific namespace (we will later join this namespace to OSM's mesh). For the purpose of this integration, we have the namespace as `dapr-demo`
+         
+           ```console
+            $ kubectl create namespace dapr-demo
```

Comment:
This is a typo here? Should this be 
 $ kubectl create namespace dapr-test
to fit with the rest of the sample?